pkg/util/logs/logs.go

```diff
@@ -4,41 +4,30 @@
 package logs
 
 import (
-	"flag"
-	"log"
-	"time"
+	"os"
+	"strconv"
 
-	"github.com/golang/glog"
-	"github.com/spf13/pflag"
-	"k8s.io/apimachinery/pkg/util/wait"
+	"github.com/rs/zerolog"
 )
 
-var logFlushFreq = pflag.Duration("log-flush-frequency", 5*time.Second, "Maximum number of seconds between log flushes")
-
-// GlogWriter serves as a bridge between the standard log package and the glog package.
-type GlogWriter struct{}
-
-// Write implements the io.Writer interface.
-func (writer GlogWriter) Write(data []byte) (n int, err error) {
-	glog.Info(string(data))
-	return len(data), nil
-}
-
-// InitLogs initializes logs the way we want for kubernetes.
+// InitLogs initializes logs with Time and Global Level of Logs set at Info
 func InitLogs() {
-	log.SetOutput(GlogWriter{})
-	log.SetFlags(0)
-	flag.Set("logtostderr", "true")
-	// The default glog flush interval is 30 seconds, which is frighteningly long.
-	go wait.Until(glog.Flush, *logFlushFreq, wait.NeverStop)
-}
-
-// FlushLogs flushes logs immediately.
-func FlushLogs() {
-	glog.Flush()
-}
-
-// NewLogger creates a new log.Logger which sends logs to glog.Info.
-func NewLogger(prefix string) *log.Logger {
-	return log.New(GlogWriter{}, prefix, 0)
+	zerolog.TimeFieldFormat = zerolog.TimeFormatUnix
+
+	// Log levels are outlined as follows:
+	// Panic: 5
```

Comment:
use an enum

```
const (
  _ = iota
  panic = iota
  fatal = iota
 // etc
)
```

Reply:
Also I would suggest just linking to the zerolog URL for the log levels and removing the rest of this comment section